src/read.py

```diff
@@ -0,0 +1,69 @@
+import re
+
+from pathlib import Path
+from bs4 import BeautifulSoup
+from nltk import tokenize, download
+from textwrap import TextWrapper
+from stqdm import stqdm
+
+import src.config as cf
+
+download('punkt', quiet=True)
+wrapper = TextWrapper(cf.MAX_CHAR_LEN, fix_sentence_endings=True)
+
+def preprocess(file):
+    input_text = BeautifulSoup(file, "html.parser").text
+    text_list = []
+    for paragraph in input_text.split('\n'):
+        paragraph = paragraph.replace('—', '-')
+        paragraph = paragraph.replace(' .', '')
+        paragraph = re.sub(r'[^\x00-\x7f]', "", paragraph)
+        paragraph = re.sub(r'x0f', " ", paragraph)
+        sentences = tokenize.sent_tokenize(paragraph)
+        
+        sentence_list = []
+        for sentence in sentences:
+            if not re.search('[a-zA-Z]', sentence):
+                sentence = ''
+            wrapped_sentences = wrapper.wrap(sentence)
+            sentence_list.append(wrapped_sentences)
+            
+        trunc_sentences = [phrase for sublist in sentence_list for phrase in sublist]
+        text_list.append(trunc_sentences)
+    text_list = [text for sentences in text_list for text in sentences]
+
+    return text_list
+
+def read_pdf(file):
+    from PyPDF2 import PdfReader
+
+    reader = PdfReader(file)
+
+    corpus = []
+    for item in stqdm(list(reader.pages), desc="Pages in pdf:"):
+        text_list = preprocess(item.extract_text())
+        corpus.append(text_list)
+
+    return corpus
+
+def read_html(file):
+    text_list = preprocess(file)
+
```

Comment:
Don't leave unnecessary spaces like this. For short functions (less than say 5 lines), no point in using any spaces. There's no rule about spacing before a return.

Reply:
Checked: removed some un-needed spaces in a few places.